source/_docs/configuration/templating.markdown

```diff
@@ -1148,6 +1148,96 @@ See: [Python regular expression operations](https://docs.python.org/3/library/re
 - Filter `value | regex_findall(find='', ignorecase=False)` will find all regex matches of the find expression in `value` and return the array of matches.
 - Filter `value | regex_findall_index(find='', index=0, ignorecase=False)` will do the same as `regex_findall` and return the match at index.
 
+## Merge action responses
+
+Using action responses we can collect information from various entities at the same time.
+Using the `merge_response` template we can merge several responses into one list.
```

Comment:
We should explain the `entity_id` will be added to the template output, and that the function will fail if there's already a key `entity_id`.

Also, let's add an example for the case where the response is a scalar.